helm/cluster-azure/values.schema.json

```diff
@@ -540,108 +623,15 @@
-                            },
-                            "disableHealthCheck": {
-                                "type": "boolean",
-                                "title": "Disable HealthChecks for the MachineDeployment"
-                            },
-                            "encryptionAtHost": {
-                                "type": "boolean",
-                                "title": "Encryption at host",
-                                "description": "Enable encryption at host for the worker nodes.",
-                                "default": false
-                            },
-                            "failureDomain": {
-                                "type": "string",
-                                "title": "Availability zone",
-                                "enum": [
-                                    "1",
-                                    "2",
-                                    "3"
-                                ]
-                            },
-                            "instanceType": {
-                                "type": "string",
-                                "title": "VM size"
-                            },
-                            "name": {
-                                "type": "string",
-                                "title": "Name",
-                                "description": "Unique identifier, cannot be changed after creation.",
-                                "maxLength": 10,
-                                "minLength": 3,
-                                "pattern": "^[-\\w\\._]+$"
-                            },
-                            "replicas": {
-                                "type": "integer",
-                                "title": "Number of nodes",
-                                "minimum": 0
-                            },
-                            "rootVolumeSizeGB": {
-                                "type": "integer",
-                                "title": "Root volume size (GB)",
-                                "exclusiveMinimum": 0
-                            }
-                        }
-                    },
-                    "default": [
-                        {
-                            "customNodeLabels": [],
-                            "customNodeTaints": [],
-                            "disableHealthCheck": false,
-                            "encryptionAtHost": false,
-                            "instanceType": "Standard_D4s_v5",
-                            "name": "md00",
-                            "replicas": 2,
-                            "rootVolumeSizeGB": 50
+                    "description": "Node pools of the cluster. If not specified, this defaults to the value of `cluster.providerIntegration.workers.defaultNodePools`.",
+                    "additionalProperties": true,
```

Comment:
This will be changed to "additionalProperties": false once I add `cluster` value to use 
```
{{- range $name, $value := .Values.global.nodePools | default .Values.cluster.providerIntegration.workers.defaultNodePools }}
```